packages/jest-worker/src/Farm.ts

```diff
@@ -44,41 +46,65 @@ export default class Farm {
     }
   }
 
-  doWork(method: string, ...args: Array<any>): Promise<unknown> {
-    return new Promise((resolve, reject) => {
-      const computeWorkerKey = this._computeWorkerKey;
-      const request: ChildMessage = [CHILD_MESSAGE_CALL, false, method, args];
+  doWork(
+    method: string,
+    ...args: Array<any>
+  ): PromiseWithCustomMessage<unknown> {
+    const customMessageListeners: Set<OnCustomMessage> = new Set();
```

Comment:
```suggestion
    const customMessageListeners = new Set<OnCustomMessage>();
```

Reply:
Okayy, I think I missed these changes which I committed at a later stage in the previous PR; when I split it. Sorry for the repeat suggestions 😅

Reply:
No worries!